### PR TITLE
Consistant configuration access.

### PR DESCRIPTION
**Pull request by lucasmichot**
Enforce consistancy for configuration access.
We shall always use getConfig for the keys which are related to the service.

### CODE REVIEW

File: src/Ovh/Provider.php [hunk 1]
```diff
@@ -22,7 +22,10 @@ class Provider extends AbstractProvider
     public static function additionalConfigKeys()
     {
         return [
+            'client_id',
+            'client_secret',
```
Comment (atymic) [inline src/Ovh/Provider.php:26]: these are default right?

File: src/Trakt/Provider.php [hunk 2]
```diff
@@ -75,6 +75,6 @@ protected function getTokenFields($code)
      */
     public static function additionalConfigKeys()
     {
-        return ['api_version'];
+        return ['api_version', 'client_id'];
```
Comment (atymic) [inline src/Trakt/Provider.php:78]: Think this is also covered by the default config items

**Review by atymic — COMMENTED** [review]
Looks good, few minor bits :)

File: src/Ovh/Provider.php (hunk 1 above)
Comment (lucasmichot) [reply src/Ovh/Provider.php]: Good catch - see 565ae93ad50d6f97970000878c940f3e0eb4b707

File: src/Trakt/Provider.php (hunk 2 above)
Comment (lucasmichot) [reply src/Trakt/Provider.php]: See 565ae93ad50d6f97970000878c940f3e0eb4b707

**Review by atymic — APPROVED** [review]

**Pull request merged**